src/lib.rs

```diff
@@ -61,6 +61,64 @@ pub use crate::util::{CastFromPrimitive, Pixel};
 
 pub use crate::frame::Frame;
 
+/// Version information
+///
+/// The information is recovered by `cargo` and `git describe` when available.
+pub mod version {
+  /// Major version component
+  ///
+  /// It is increased every time a release presents a incompatible API change.
+  pub fn major() -> usize {
+    env!("CARGO_PKG_VERSION_MAJOR").parse().unwrap()
+  }
+  /// Minor version component
+  ///
+  /// It is increased every time a release presents new functionalities are added
+  /// in a backwards-compatible manner.
+  pub fn minor() -> usize {
+    env!("CARGO_PKG_VERSION_MINOR").parse().unwrap()
+  }
+  /// Patch version component
+  ///
+  /// It is increased every time a release provides only backwards-compatible bugfixes.
+  pub fn patch() -> usize {
+    env!("CARGO_PKG_VERSION_PATCH").parse().unwrap()
+  }
+
+  /// Version information as presented in CARGO_PKG_VERSION
+  ///
+  /// Can be parsed by [semver](https://crates.io/crates/semver).
+  pub fn short() -> String {
+    env!("CARGO_PKG_VERSION").to_string()
+  }
+
+  /// Version information as presented in CARGO_PKG_VERSION followed by the
+  /// short commit hash if present.
```

Comment:
This documentation isn't super useful to API users who just want to know what sort of string it outputs; it relies on specific Rust env knowledge. Same for the others.

Reply:
I assume that this corresponds to the `version` line in the `[package]` category in Cargo.toml. I think it may be clearer in those terms, as it is easier to locate for those unfamiliar with Cargo.